core/src/common/lwm2m_result.c

```diff
@@ -60,3 +61,30 @@ int AwaLwm2mResult_ToAwaError(AwaLwm2mResult result, int defaultError)
         return defaultError;
     }
 }
+
+int AwaLwm2mResult_ToAwaLwm2mError(AwaLwm2mResult result)
```

Comment:
This isn't required any more - the two error spaces don't map to each other, they are separate. Best not to include unused (and untested) code.